.babelrc

```diff
@@ -0,0 +1,35 @@
+{
+  "presets": [
+    [
+      "env",
```

Comment:
Please add `babel-preset-env` as dependency

Reply:
this comes from https://github.com/webpack-contrib/webpack-defaults/blob/master/src/tasks/babel.js. if/when that's updated there, this will automagically be added by way of applying a new webpack-defaults version to the repo.